api_types.ts

```diff
@@ -996,14 +996,23 @@ export interface ExecutionContext {
 
   /**
    * If this invocation is a part of a crawling execution, like in Coda Brain, then this ID will be provided
-   * to all invocations. That includes invocations of sync `execute` and `executeGetPermissions`, as well as
-   * dynamic table features like `listDynamicUrls`, `getSchema`, and `getName`.
+   * to all invocations as incremental execution id. That includes invocations of sync `execute` 
+   * and `executeGetPermissions`, as well as dynamic table features like `listDynamicUrls`,
+   * `getSchema`, and `getName`.
    *
    * TODO(patrick): Unhide this
    * @hidden
    */
   readonly executionId?: string;
 
+  /**
+   * Provided by a crawling execution, like in Coda Brain, this ID will be the full execution id.
+   *
+   * TODO(ebo): Unhide this
+   * @hidden
+   */
+  readonly fullExecutionId?: string;
```

Comment:
If we're using "full sync" to mean an ingestion, then it's odd to use "full/incremental" here the way we do internally. I could see this being `baseExecutionId` perhaps...

But then again, maybe this is great as it is, and we should instead figure out a better word for "full sync". How about we expose the word "ingestion" in the SDK comments @jonathan-codaio @ekoleda-codaio ?

Reply:
Relatedly, this would mean "fullSyncSuggestedValue" should maybe become "ingestionSuggestedValue" cc @pardeep-codaio